scripts/getStatus.js

```diff
@@ -50,11 +52,10 @@ function writeContent(content) {
   fs.writeFileSync(logFilepath, content);
 }
 
-//runs the linter on a given swagger spec.
-async function runLinter(readme) {
-    let cmd = 'autorest ' + readme + ' --azure-validator=true --message-format=json';
+//runs the command on a given swagger spec.
+async function runCmd(linterCmd) {
```

Comment:
Can you change this to `cmd` and replace the uses of `linterCmd` with `cmd`?

Reply:
Done...

Reply:
Fixed...thanks.